src/pytest_allure_spec_coverage/plugin.py

```diff
@@ -58,6 +54,18 @@ def pytest_register_spec_collectors(collectors: CollectorsMapping) -> None:
     collectors["sphinx"] = SphinxCollector
 
 
+@dataclasses.dataclass(eq=False)
```

Comment:
Import `dataclasses.dataclass` for consistency.

Reply:
fix